torchvision/prototype/transforms/functional/_color.py

```diff
@@ -18,10 +19,12 @@ def adjust_brightness_image_tensor(image: torch.Tensor, brightness_factor: float
     if brightness_factor < 0:
         raise ValueError(f"brightness_factor ({brightness_factor}) is not non-negative.")
 
-    _FT._assert_channels(image, [1, 3])
+    c = image.shape[-3]
+    if c not in [1, 3]:
+        raise TypeError(f"Input image tensor permitted channel values are {[1, 3]}, but found {c}")
```

Comment:
```suggestion
        raise TypeError(f"Input image tensor permitted channel values are 1 or 3, but found {c}")
```

Reply:
This is the error we have in all other methods, check the rest of the methods.

Reply:
Didn't see that sorry. Yes, we should change it everywhere. Dynamic string creation does not make sense if the value is static. You can do it here, but I'm also ok to do that in a follow-up.

Reply:
All good. I did it here for all.